packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.base.tsx

```diff
@@ -167,6 +168,7 @@ export class HoverCardBase extends React.Component<IHoverCardProps, IHoverCardSt
         ref={this._hoverCard}
         aria-describedby={setAriaDescribedBy && isHoverCardVisible ? hoverCardId : undefined}
         data-is-focusable={!Boolean(this.props.target)}
+        {...(hostNativeProps && { ...getNativeProps(hostNativeProps!, divProperties) })}
```

Comment:
Same question here... what is your use case for needing these?

Reply:
To pass accessibility props for outer host element